src/stories/Forms/Select/Select.tsx

```diff
@@ -281,7 +301,7 @@ export interface SelectProps extends GlobalInputProps {
    * Option components to be rendered.
    */
   children: React.ReactNode;
-  showItems?: number;
+  multiselect?: boolean;
```

Comment:
Maybe we should add a comment about what format multiselect returns values on change event? Example: "Returns a comma sepparated list with selected elements"?